data/ui_scripts/custom_depot/dkjson.lua

```diff
@@ -0,0 +1,748 @@
+-- Module options:
+local always_use_lpeg = false
```

Comment:
Why are you adding another json lib?

Reply:
Because you are trying to encode a sparse array like the error says, you should convert the array keys to string which will fix the issue.